.travis.yml

```diff
@@ -10,23 +10,23 @@ os:
   - osx
 
 go:
-- "1.12"
+- "1.13"
```

Comment:
Please revert this change.

Reply:
But the goal is to upgrade the project to work/use go version 1.13. So, why the Travis would test check it with 1.12?